scripts/mdpow-solvationenergy

```diff
@@ -105,20 +105,59 @@ def run_gsolv(solvent, directory, **kwargs):
 
     gsolv = load_gsolv(solvent, directory, permissive=kwargs.pop('permissive',False),)
 
+    if not hasattr(gsolv, 'start'):
+        gsolv.start = kwargs.pop('start', 0)
+    if not hasattr(gsolv, 'stop'):
+        gsolv.stop = kwargs.pop('stop', None)
+    if not hasattr(gsolv, 'SI'):
+        kwargs.setdefault('SI', True)
+    else:
+        kwargs.setdefault('SI', gsolv.SI)
+
+    # for this version. use the method given instead of the one in the input cfg file
+    gsolv.method = kwargs.pop('method', 'MBAR')
+    estimator = kwargs.pop('estimator', 'alchemlyb')
+
     # do the analysis (only use keywords that are needed for FEP.analysis())
     akw = {'stride': kwargs.pop('stride', None), 'force': kwargs.pop('force', False)}
+    if not estimator in ('mdpow', 'alchemlyb'):
+        errmsg = "estimator = %r is not supported, must be 'mdpow' or 'alchemlyb'" % estimator
+        logger.error(errmsg)
+        raise ValueError(errmsg)
     if akw['force']:
```

Comment:
The logic between force and not finding the data should be combined; at the moment there's too much code duplication. The logic should be simple:
```
if force or don't have data:
     analyze

# now we have data
calculate DG
```